src/bin/fullnode.rs

```diff
@@ -66,7 +66,7 @@ fn main() {
                 .help("Custom RPC port for this node"),
         ).get_matches();
 
-    let sigverify = !matches.is_present("nosigverify");
+    let sigverify = matches.is_present("nosigverify");
```

Comment:
prolly should rename variable to "nosigverify"

Reply:
Renamed the variable to sigverify